migrations/20161201122820_users.js

```diff
@@ -0,0 +1,15 @@
+
+exports.up = function(knex, Promise) {
```

Comment:
I see you have `bluebird` in npm dependencies, is it needed here?
or can it work without it?
Usually if `Promise` is referenced, there needs to be `import Promise from 'bluebird'` or else it would use default es6 Promise which might also work here.